### ISSUE TITLE
Opening project after created

### ISSUE THREAD

**Issue by banillie** [yulqen/dbasik#6]
After creating project receiving an error when try to access via http://127.0.0.1:8000/register/project/[project slug]. 

Not an issue for admin page. Project details can be accessed there. 

Only an issue when after project has been created. Create new project works fine

**Comment by yulqen** [comment]
Yep - expected. Lots of these redirects and confirmation links will be broken. Don't think I ever implemented them. That said, if this doesn't look like the reason, paste the exception in here and I can take a look.
